### PR TITLE
Replace generic hermit crab icon with oasis emoji as app icon

### PR DESCRIPTION
**Pull request by alizehctl**

### CODE REVIEW

**Comment by christianbundy** [comment]
This looks great! My only concern: what font is that, and is it open source? I think we'd be safe to use Noto Emoji, because I know it has an open license, but I'm not sure about other fonts.

**Comment by aadilayub** [comment]
> This looks great! My only concern: what font is that, and is it open source? I think we'd be safe to use Noto Emoji, because I know it has an open license, but I'm not sure about other fonts.

Good question! It's Noto Emoji.

**Comment by christianbundy** [comment]
Sweet! LGTM.

**Comment by christianbundy** [comment]
Hmm, it looks like it's failing on the ICNS: https://travis-ci.com/github/fraction/oasis-desktop/jobs/328239556

I don't know why that would be.

**Comment by aadilayub** [comment]
hmm... yeah I found ICNS conversion to be super finicky. It took me a few tries to generate a file that wasn't corrupt. Let me try again. How does this work? Do  I make another PR?

**Comment by christianbundy** [comment]
You can just push another commit to this branch and it'll re-run!

**Comment by aadilayub** [comment]
argh, it's still failing. I guess I'll have to read up a bit more about .icns and how to properly create one.
